Model/Api/OrderManagement.php

```diff
@@ -253,11 +260,24 @@ private function handleCartCreateApiCall()
             );
         }
 
-        $cart = $this->cartHelper->createCartByRequest($request);
-        $this->response->setHttpResponseCode(200);
-        $this->response->setBody(json_encode([
-            'status' => 'success',
-            'cart' => $cart,
-        ]));
+        try {
+            $cart = $this->cartHelper->createCartByRequest($request);
+            $this->response->setHttpResponseCode(200);
+            $this->response->setBody(json_encode([
+                'status' => 'success',
+                'cart' => $cart,
+            ]));
+        } catch (\Exception $e) {
+            $error_message = $e->getMessage();
+            error_log($e->getCode());
+            error_log($e->getMessage());
+            if ($error_message == 'The requested qty is not available') {
```

Comment:
I could not find this error message in Magento code base. The error handling in this file does not seem to be done right.